Library/Formula/libfixposix.rb

```diff
@@ -0,0 +1,42 @@
+class Libfixposix < Formula
+  env :std
+  desc "Thin wrapper over POSIX syscalls"
+  homepage "https://github.com/sionescu/libfixposix"
+  url "https://github.com/sionescu/libfixposix.git", :tag => "v0.3.0", :revision => "33d0a3365e9f911752a79229a2759a820892713c"
```

Comment:
No submodule required. I haven't found any available tarball. Could build one but don't know where to put it. Is there a way?